File: PSW-projekat/PZL-Angular/src/app/modules/hospital/model/medicine.ts
export class Medicine {
    id: number;
    deleted: boolean;
    name: string = "";
    quantity: number = 0;

    public constructor(id: any, deleted: any, name: any, quantity: any) {
        this.id = id;
        this.deleted = deleted;
        this.name = name;
        this.quantity = quantity;
    }
}

File: PSW-projekat/hospital-be-main/src/HospitalLibrary/Core/Model/Medicine.cs
namespace HospitalLibrary.Core.Model
{
    public class Medicine: BaseModel
    {
        public string Name { get; set; }
        public int Quantity { get; set; }

        public Medicine( int id, bool deleted, string name, int quantity)
        {
            Id = id;
            Deleted = deleted;
            Name = name;
            Quantity = quantity;
        }
    }
}
